src/north/MongoDB/MongoDB.class.js

```diff
@@ -56,10 +56,11 @@ class MongoDB extends ApiHandler {
     this.logger.trace(`MongoDB handleValues() call with ${values.length} values`)
     try {
       await this.makeRequest(values)
-      this.statusData['Last handled values at'] = new Date().toISOString()
-      this.statusData['Number of values sent since OIBus has started'] += values.length
-      this.statusData['Last added point id (value)'] = `${values[values.length - 1].pointId} (${JSON.stringify(values[values.length - 1].data)})`
-      this.updateStatusDataStream()
+      this.updateStatusDataStream({
```

Comment:
Most handleValues return the same information should the sse call be  in the parent class ?

Reply:
I agree, but I don't think it should be done here, rather in the cache refactor in PR #1797